### ISSUE TITLE
UnusedImportRule + `==` broken with Xcode 11.5

### ISSUE THREAD

**Issue by keith** [realm/SwiftLint#3256]
When using the UnusedImportRule to identify unused imports when only an operator from a module is used, it appears SourceKit no longer returns the module defining the Equatable operator override and instead returns that it comes from Swift.

In our example we have:

```swift
public func == (lhs: CLLocationCoordinate2D, rhs: CLLocationCoordinate2D) -> Bool {
...
}
```

And then a file uses this directly. It appears that SwiftLint still does the expected cursorinfo request on the `==` at the callsite, but it now returns Swift instead of our custom module. This is probably something we should report upstream with a good repro case, but I wanted to file here too as a marker.

**Comment by keith** [comment]
Still relevant
